source/dem/dem_container_manager.cc

```diff
@@ -250,6 +250,24 @@ DEMContainerManager<dim>::execute_particle_particle_broad_search(
     }
 }
 
+template <int dim>
+void
+DEMContainerManager<dim>::execute_particle_particle_broad_search(
+  dealii::Particles::ParticleHandler<dim> &particle_handler,
+  const DisableParticleContact<dim> &      disable_particle_contact_object,
+  const bool                               has_periodic_boundaries)
+{
+  particle_particle_broad_search_object.find_particle_particle_contact_pairs(
+    particle_handler, *this, disable_particle_contact_object);
+
+  if (has_periodic_boundaries)
+    {
+      particle_particle_broad_search_object
+        .find_particle_particle_periodic_contact_pairs(
```

Comment:
I would need to pass the `has_periodic_boundaries` again, it seems a lot. Also, this class use this pattern in other functions also for the has_floating_mesh or walls. It could be refactor, but not in this PR in my opinion

Reply:
Agreed. We could add it to the to-do list